### PR TITLE
added sitemap.xml file

### PR DESCRIPTION
**Pull request by AnirudhPanda**
Generated  **sitemap.xml**  file and added it to the root as well as in robots.txt but further ownership required to add it on the google search console.

### CODE REVIEW

File: public/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+      xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+      xsi:schemaLocation="http://www.sitemaps.org/schemas/sitemap/0.9
+            http://www.sitemaps.org/schemas/sitemap/0.9/sitemap.xsd">
+<!-- created with Free Online Sitemap Generator www.xml-sitemaps.com -->
```
Comment (anushbhatia) [inline public/sitemap.xml:7]: Remove this

File: public/robots.txt [hunk 2]
```diff
@@ -1,3 +1,4 @@
 # https://www.robotstxt.org/robotstxt.html
```
Comment (anushbhatia) [inline public/robots.txt:1]: Also try removing this

**Review by anushbhatia — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Fix the merge conflicts please

File: public/robots.txt (hunk 2 above)
Comment (AnirudhPanda) [reply public/robots.txt]: https://github.com/smaranjitghose/doc2pen/pull/1008/commits/0d164230a69e4a7bc04fc134664109c9150cf758

File: public/sitemap.xml (hunk 1 above)
Comment (AnirudhPanda) [reply public/sitemap.xml]: https://github.com/smaranjitghose/doc2pen/pull/1008/commits/0d164230a69e4a7bc04fc134664109c9150cf758

**Comment by AnirudhPanda** [comment]
> Fix the merge conflicts please

I don't know why but merge conflicts isn't showing to me

**Comment by anushbhatia** [comment]
conflict is in package-lock.json. Create a new PR that's best

**Comment by AnirudhPanda** [comment]
> conflict is in package-lock.json. Create a new PR that's best

Okay

**Pull request closed without merging**